gateway/services/upload.go

```diff
@@ -11,6 +11,7 @@ import (
 var UploadURL = config.UPLOAD_SERVICE
 
 const UploadFormat string = "JSON"
+const RawFormat string = "RAW"
```

Comment:
Change RawFormat to UploadFormat, since that is the format used to actually upload files via a PUT request, and change UploadFormat to InfoFormat, since that is used only to query information. Update the usage as well.